### DISCUSSION TITLE
How to redirect to the newly created item instead of the list of items?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2270] [Ideas]
Hello,

I have a form. The form button has an action navigate that lead the the list view. But I want it to redirect directly to the detail vue, is it possible?

As the detail vue requiered the id...

Thank you

**Comment by mjashanks** [comment]
I know exactly what you mean! Unfortunately, it's not possible, as the `_id` is not returned to the action.

Have turned this into an idea, as I would also like to have this.

**Comment by oswaldB** [comment]
Ok thank you.

I am rethinking my process then.

Have a nice day
